catalog/app/website/components/BucketGrid/BucketGrid.js

```diff
@@ -97,43 +75,99 @@ const useStyles = M.makeStyles((t) => ({
   },
 }))
 
+function Bucket({ bucket, onTagClick, tagIsMatching }) {
+  const b = bucket
+  const classes = useBucketStyles()
+  const { urls } = NamedRoutes.use()
+
+  return (
+    <div className={classes.bucket}>
+      <div>
+        <div className={classes.shared}>
+          {b.collaborators?.length && (
```

Comment:
```suggestion
          {!!b.collaborators?.length && (
```
otherwise this will render `0` when there's no collaborators